rca/people/models.py

```diff
@@ -1,65 +1,172 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 from modelcluster.fields import ParentalKey
-from wagtail.admin.edit_handlers import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    ObjectList,
+    PageChooserPanel,
+    StreamFieldPanel,
+    TabbedInterface,
+)
+from wagtail.core.fields import RichTextField, StreamField
 from wagtail.images.edit_handlers import ImageChooserPanel
 
-from rca.utils.models import BasePage
+from rca.utils.blocks import AccordionBlockWithTitle, GalleryBlock, LinkBlock
+from rca.utils.models import BasePage, RelatedPage
 
 
-class SocialMediaProfile(models.Model):
-    person_page = ParentalKey("StaffPage", related_name="social_media_profile")
-    site_titles = (("twitter", "Twitter"), ("linkedin", "LinkedIn"))
-    site_urls = (
-        ("twitter", "https://twitter.com/"),
-        ("linkedin", "https://www.linkedin.com/in/"),
+class AreaOfExpertise(models.Model):
+    title = models.CharField(max_length=128)
+
+    def __str__(self):
+        return self.title
+
+
+class StaffPageAreOfExpertisePlacement(models.Model):
+    page = ParentalKey("StaffPage", related_name="related_area_of_expertise")
+    area_of_expertise = models.ForeignKey(
+        AreaOfExpertise, on_delete=models.CASCADE, related_name="staff"
```

Comment:
For consistency, 'related_staff' might make a better `related_name` value here

Reply:
fixed in https://github.com/torchbox/rca-wagtail-2019/pull/426/commits/88b0b258220d5a421b8f5794d9ab35fc975fee2c